time start_time
float64 duration
int64 index
